.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+  "parserOptions": {
+    "sourceType": "module",
```

```suggestion
    "sourceType": "module"
```
Ups, my bad - my change suggestion left a comma where it shouldn't be